geoh5py/objects/surveys/electromagnetics/tipper.py

```diff
@@ -170,18 +167,27 @@ def default_units(self) -> list[str]:
         """
         return self.__UNITS
 
-    @property
-    def tx_id_property(self) -> ReferencedData | IntegerData | None:
-        """
-        Data link between the receiver and transmitter object.
+    def _format_transmitter_ids(self, values, attributes):
```

Comment:
See other comment about the same function

Reply:
We may need to clarify this in person